### PR TITLE
Fix image links for plugins by @rnkn

### PR DESCRIPTION
**Pull request by rnkn**
I moved a domain and forgot about these image links.

Seems like you've been doing rebase merges in the past, which has made merging this a bit messy for you now... but I'll leave that up to you.... rebase = no-no.

### CODE REVIEW

**Comment by rnkn** [comment]
Closes #1290

**Comment by rnkn** [comment]
Added a few other fixes to these three plugins. Sorry for the mess.

File: Network/sent-received.1s.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ TMPFILE="${TMPDIR}/bitbar-sent-received"
 
 if [[ -r "${TMPFILE}" ]]
 then
-    read -ra offsets < "${TMPFILE}"
+    read offsets < "${TMPFILE}"
```
Comment (tresni) [inline Network/sent-received.1s.sh]: The `-r` here are not the same as the `-r` in the `if` statement.  In this case it does 'raw' handling of input vs checking if the file is readable

File: System/launch-agents.10s.sh [hunk 2]
```diff
@@ -16,43 +16,41 @@
 
 # ==============================================================================
 
-launchctl=$(which launchctl)
-defaults=$(which defaults)
 open=$(which open)
```
Comment (tresni) [inline System/launch-agents.10s.sh]: https://travis-ci.org/matryer/bitbar-plugins/builds/640097794#L1063

File: Network/sent-received.1s.sh (hunk 1 above)
Comment (tresni) [reply Network/sent-received.1s.sh]: https://travis-ci.org/matryer/bitbar-plugins/builds/640097794#L1011

**Comment by rnkn** [comment]
Thanks for the followup. I’m not using BitBar anymore, but this PR is just to fix broken image URLs. Feel free to use the image URLs (or images) and discard the rest, or fork and make changes as you see fit.

**Comment by tresni** [comment]
Closed with 7b2eeecf9b387b5f893e7931cfb3c130170475b2

**Pull request closed without merging**